packages/utils/test/baggage.test.ts

```diff
@@ -129,4 +160,70 @@ describe('Baggage', () => {
       expect(mergeAndSerializeBaggage(baggage, headerBaggageString)).toEqual(outcome);
     });
   });
+
+  describe('parseBaggageSetMutability', () => {
+    it.each([
```

Comment:
I put this in abhi's original PR but at what point can I convince you to write non-table tests here? :D This is so unwieldy

Reply:
I do remember the comment ;D 

So I think that for some of the functions I could change it because the table has a length of 2. So in that case I think it could be changed to two individual tests. 
However, for the `parseBaggageSetMutability` strings, I prefer the parametrized/tabled tests because I would essentially just copy the same code over and over again for different input. So this spares us a little duplicated code.
I do agree that at first this style of test writing is a little harder to read in the first place but I don't think that the alternative (i.e. 6 very similar/identical tests) isn't better either.  

On a side note: Having worked a lot with testing during my Master's thesis, I don't like parametrized tests because they're harder to parse and to handle than single-input unit tests if you write testing tools. But that's about it. Generally I think, parametrized tests are a good thing to have, also as a first step if we ever want to try out more sophisticated testing approaches. I'm thinking of e.g. property-based testing where we would just define predicates how the SUT should behave and the input is generated randomly. But yeah, just some random thoughts on testing there, pretty off-topic 😅

Reply:
Sounds good! Thanks for the insight ^^